tests/testthat/test-sas-results-part-1.R

```diff
@@ -0,0 +1,398 @@
+test_that("Posterior summaries for probabilities of
+          DLT (2-parameter logistic model) and recommended doses (NCRMLoss):
+          crmPack vs. SAS - Example 1", {
+  set.seed(0504201914)
+  mcmc_options <- McmcOptions(
+    burnin = 5000,
+    step = 2,
+    samples = 1000000,
+    rng_kind = "Wichmann-Hill",
+    rng_seed = 1
+  )
+
+  dose_grid_sc1 <- c(10, 20, 35, 50, 65, 80, 90, 100)
+  tab_sc1 <- vector("list")
```

Comment:
thanks for splitting it up @marlenesg ! nice. Now that we have that, no need for lists anymore I think here - it will be easier to read when you just e.g. use `data <- Data(...)` and similarly for the other inputs.